File: Java/Lista_04(Vetores)/Mais_Velho.java
import java.util.Scanner;

public class Mais_Velho {
    
    public static void main(String[] args){

        Scanner s = new Scanner(System.in);

        System.out.print("Quantas pessoas voce vai digitar? ");
        int n = s.nextInt();

        String nome[] = new String[n];
        int idade[] = new int[n];

        for(int i=0;i<nome.length;i++){
            System.out.println("Dados da " + (i+1) + "a pessoa:");
            System.out.print("Nome: ");
            nome[i] = s.next();
            System.out.print("Idade: ");
            idade[i] = s.nextInt();
        }

        int maior = idade[0];
        String maisVelho = nome[0];

        for(int i=0;i<nome.length;i++){
            if(idade[i]>maior){
                maior = idade[i];
                maisVelho = nome[i];
            }
        }

        System.out.print("PESSOA MAIS VELHA: " + maisVelho);

        s.close();

    }
}

File: Python/Lista_04(Vetores)/Mais_Velho.py
n = int(input("Quantas pessoas voce vai digitar? "))

nome = [0 for x in range(n)]
idade = [0 for x in range(n)]

for i in range(0,n):
    print(f"Dados da {i+1}a pessoa:")
    nome[i] = input("Nome: ")
    idade[i] = int(input("Idade: "))

maisVelho =  nome[0]
id = idade[0]

for i in range(0,n):
    if idade[i]>id:
        id = idade[i]
        maisVelho = nome[i]

print(f"PESSOA MAIS VELHA: {maisVelho}")